src/common/meta/src/key/flow/flow_name.rs

```diff
@@ -45,6 +51,10 @@ impl<'a> FlowNameKey<'a> {
         FlowNameKey(FlowScoped::new(inner))
     }
 
+    pub fn range_start_key(catalog: &str) -> String {
+        format!("{}/{}/{}/", FLOW_PREFIX, FLOW_NAME_KEY_PREFIX, catalog)
```

Comment:
Let's follow the style
```suggestion
        let inner = BytesAdapter::from(FlowNameKey::prefix(catalog).into_bytes());
```